2021-03-07 23:27:27,305:__main__:Tried to divide by zero
Traceback (most recent call last):
  File "/Users/raj.kumar/Desktop/pythonProject1/Mislenious/Logs--logging.py", line 140, in divide
    result = x / y
ZeroDivisionError: division by zero
